libraries/registration-api-client/src/main/java/no/dcat/client/registrationapi/RegistrationApiClient.java

```diff
@@ -51,6 +51,7 @@
         } catch (Exception e) {
             logger.info("Error. Cannot read published apis from registration. {}", e.getMessage());
             logger.debug("Error. Stack trace", e);
+            return null;
```

Comment:
en alternativ mulighet kunne være å bare ikke catch error, og la de bubble gjennom.

Reply:
Godt poeng. skal tenke på det

Reply:
Tenker dette blir et lite forbedringspotensiale for senere